### ISSUE TITLE
Migrate repo to acmcsufoss/acmcsuf.com

### ISSUE THREAD

**Issue by amyipdev** [acmcsufoss/acmcsuf.com#1107]
### What's your idea?

As much as we and I love Ethan, he won't be around forever; both for future maintenance reasons, and for professionalism, we should work on migrating this repo to acmcsufoss/acmcsuf.com.

**Comment by EthanThatOneKid** [comment]
Thanks for the feedback, @amyipdev :)

acmcsuf.com is currently hosted on my free Netlify account. Once we identify a CI/CD solution for our GitHub organization, I'll be happy to migrate the repository to a more collaborative environment.

**Comment by amyipdev** [comment]
We're hosting on _free_ Netlify? Wow, didn't realize that at all - I thought we were hosting on Cirno. 

Could migrating to Cirno be an option?

**Comment by EthanThatOneKid** [comment]
> Could migrating to Cirno be an option?

If I recall correctly, Cirno is an AWS EC2 instance. We previously had an automated deployment process for Cirno using GitHub Actions/Workflows. I'm uncertain about its current status and if it's still operational.

**Comment by amyipdev** [comment]
Yes, Cirno is EC2, and yes that process still works - I actually am the one who oversees it now.

**Comment by EthanThatOneKid** [comment]
Sounds good to me. Then once we completely migrate acmcsuf.com to Cirno, I’ll move forward with transferring the GitHub repository to @acmcsufoss!

**Comment by GaballaGit** [comment]
Repo has been migrated to acmcsufoss/acmcsuf.com utilizing cloudflare for deployment on 1/10/26.
